.github/workflows/Testbase.yml

```diff
@@ -18,9 +18,9 @@ jobs:
       QT_DEBUG_PLUGINS: 1
     steps:
       - name: Set up Python ${{ inputs.python }}
-        uses: actions/checkout@v4
+        uses: actions/checkout@v4.2.2
```

Comment:
As far as I know, @v4 resolves always to the most recent minor / patch version, so it will use v4.2.2 now and later v4.2.3, once that is released.

So, no need to change it here (unless you want to pin that specific version).